features/270-next-skycli/commands.md

```diff
@@ -283,7 +283,7 @@ Run kubectl config use-context skygear-skygear-production-myapp to switch to the
 
 `skycli app delete-k8s-credentials`
 
-Delete the credentials remotely if exists. Then delete the config from the machine's kubeconfig.
+Delete the config from the local machine's kubeconfig.
```

Comment:
Why not delete the credentials remotely too?

Reply:
The TokenRequest is actually a JWT token. [client-go does not expose a high level API to delete TokenRequest.](https://godoc.org/k8s.io/client-go/kubernetes/typed/core/v1#ServiceAccountExpansion)

Reply:
I suppose we can delete the service account, and it would have the same effect?

Reply:
I have thought about it. See the concerns in the commit message.